src/core/qgseditformconfig.cpp

```diff
@@ -272,7 +272,7 @@ void QgsEditFormConfig::readXml( const QDomNode& node )
   }
 
   QDomNode editFormInitCodeSourceNode = node.namedItem( QStringLiteral( "editforminitcodesource" ) );
-  if ( !editFormInitCodeSourceNode.isNull() || ( !editFormInitCodeSourceNode.isNull() && !editFormInitCodeSourceNode.toElement().text().isEmpty() ) )
+  if ( !editFormInitCodeSourceNode.isNull() && !editFormInitCodeSourceNode.toElement().text().isEmpty() )
```

Comment:
as far as I can see it just removes the redundant `isNull()` check, so yes, I don't see any functional difference but the new version is cleaner.